### PR TITLE
local-gadget: Add oomkill trace gadget

### LINKED ISSUE #890: local-gadget: Run trace/oomkill gadgets using tracers instead of the Tracer CR
## Current situation

`local-gadget` does not support `oomkill` trace gadget.

## Ideal future situation

Add `local-gadget trace oomkill` by:

- Resuing the command definition (description and flags) already created for `kubectl-gadget`.
- Verifying no regression tests are introduced to `kubectl-gadget`
- Updating the documentation.

The PRs https://github.com/kinvolk/inspektor-gadget/pull/896 and https://github.com/kinvolk/inspektor-gadget/pull/937 added support for `bind`, `exec` and `tcp` trace gadgets, so they can be used as a reference to solve this issue.

See https://github.com/kinvolk/inspektor-gadget/issues/864 for further details.

### PR DESCRIPTION
**Pull request by blanquicet**
Fixes #890

### CODE REVIEW

File: integration/local-gadget/trace_oomkill_test.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	. "github.com/kinvolk/inspektor-gadget/integration"
+	oomkillTypes "github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/oomkill/types"
+)
+
+func TestTraceOOMKill(t *testing.T) {
+	t.Parallel()
+	ns := GenerateTestNamespaceName("test-trace-oomkill")
+
+	traceOOMKillCmd := &Command{
+		Name:         "StartOomkilGadget",
```
Comment (mauriciovasquezbernal) [inline integration/local-gadget/trace_oomkill_test.go:30]:
```suggestion
		Name:         "StartOomkillGadget",
```

**Review by mauriciovasquezbernal — APPROVED** [review]
LGTM. I tested with the yaml you're using for the integration test and it worked fine.

**Review by mauriciovasquezbernal — CHANGES_REQUESTED** [review]
Please update imports names before merging. (https://github.com/inspektor-gadget/inspektor-gadget/pull/1040).

**Comment by blanquicet** [comment]
Rebased and updated.

**Review by mauriciovasquezbernal — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**